samples/serverReports/src/sql/waits_paul_randal.sql

```diff
@@ -0,0 +1,76 @@
+-- SOURCE: https://www.sqlskills.com/blogs/paul/wait-statistics-or-please-tell-me-where-it-hurts/
+-- Use for a demo purpose only. This query is not built-in to any product.
```

Comment:
since we are bundling this as an extension, is it correct to say 'This query is not built-in to any product'

Reply:
I change to "This query is not built-in to any product" Will verify with Eric

---
In reply to: [176588399](https://github.com/Microsoft/sqlopsstudio/pull/960#discussion_r176588399) [](ancestors = 176588399)